ui/src/message_popup/questions.js

```diff
@@ -297,6 +297,113 @@ export const motivationQuestions:[Question] = [
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 307,
+    text: 'Students are working in small groups of three or four. You scan and notice that in one group only one student is doing all the work while the other two are drawing and in another group one of the students keeps calling another student "stupid" and telling the student that his/her ideas, "suck".',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 308,
+    text: 'You have started a well planned launch of a lesson where just about every word and idea is critical to the student entry into the problem solving part of the lesson. A student waves his/her hand and asks to go to the bathroom.',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 309,
+    text: 'It is group work time, and students are working on a problem solving protocol on chart papers.  One group calls out, “who has a green marker?”  Across the room--the green marker flies, successfully, from one group to another.',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 310,
+    text: 'It\'s rug time. You transition the class to the rug. One student refuses to leave his seat and starts yelling out negative comments, "I hate this class. The work we do is stupid."',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 311,
+    text: 'The lesson is going along smoothly. You ask a question and calls on a student, as the student begins to respond, another student then yells out, "You NEVER call on me!" and rips their paper up.',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 313,
+    text: 'The class is actively engaged in a discussion about a time they felt grown up, and as you scan the room, you notice a student has their head down and is staring into space.',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 314,
+    text: 'Students have just returned to class from special and have started the routine of independent reading. You are passing out the snack for the day - bananas. As you reach one student, she asks, "What can we have if we don\'t like bananas?" You say, "Bananas are the only choice. It\'s bananas or you can wait until lunch." The student yells out "WHAT?! I don\'t like bananas," walks out of her seat, rolls up half the rug, and then starts slamming the door repeatedly.',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 315,
+    text: 'The class is lining up for specials in line order. One of your students decides cut in front of another student. You hear the other student ask her to kindly move but she refuses with a smirk on her face. You tell the student she needs to return to her line spot, but she continues to stand there silently. Your student who was cut starts to stomp her feet and yell "That\'s not fair that she gets to cut!"',
```

Comment:
Typo here: "One of your students decides cut in front of another student." => "One of your students decides _**to**_ cut in front of another student."